test-openjdk/src/test/java/com/redhat/qe/openjdk/image/AbstractDockerImageTest.java

```diff
@@ -57,7 +57,14 @@ public abstract class AbstractDockerImageTest extends OpenJDKTestParent {
 			"jps", "jrunscript", "jshell", "jstack", "jstat",
 			"jstatd", "keytool", "rmiregistry", "serialver"
 	};
-
+	public static final String[] DEFAULT_JAVA_21_UTILITIES = new String[]{
+			"alt-java", "jarsigner", "javac", "javap", "jconsole",
```

Comment:
I see alt-java in the runtime images (=headless non-devel RPM) but not the builders, oddly

Reply:
Let me expand on that. In the builder images, alt-java is in /usr/bin but not in any of /usr/lib/jvm/*/bin. I haven't traced in the test where this list of binaries is checked against. My entire comment may be a waste of time

Reply:
also FWIW jcmd is now in the runtime images, and didn't used to be

Reply:
This testsuite currently does not run against the runtimes due to the historic nature of the tests.